FRANKDOC.md

```diff
@@ -0,0 +1,86 @@
+# Frank!Doc user manual for contributors
+
+As a contributor to the Frank!Framework, you write JavaDoc comments that are used to generate the Frank!Doc. This document explains how your comments appear in this user documentation. It also explains some Java annotations that you need to maintain the Frank!Doc.
+
+## What is the Frank!Doc?
+
+From a user perspective, the Frank!Doc consists of two parts. First, Frank developers use the file `FrankConfig-strict.xsd` that is generated by the frankDoc doclet (see [CONTRIBUTING](./CONTRIBUTING.md)). Frank developers typically get this file under the name `frankdoc.xsd`. They include a reference to this XML schema in the XML Frank configs they write. This gives them autocomplete functionality and tooltip help in their text editor.
+
+The second part is a [web application](https://ibis4example.ibissource.org/iaf/frankdoc) with reference information about the XML-based language of Frank configurations. Both parts are shown here now to introduce you to the Frank!Doc.
+
+#### XML schema for Frank developers
+
+A Frank developer uses the XML schema when she writes a Frank config. She may start writing a config in Visual Studio Code as shown below:
```

Comment:
Het werkt een beetje op mijn zenuwen als je overal 'she' gebruikt waar je 'he' zou verwachten.

Reply:
Ik heb het aangepast. Je hebt geluk dat er geen feministen in het core team zitten. Ik heb een vrouwelijke collega-ontwikkelaar gehad in een vorige baan. Zij wilde ook liever geen "she" in plaats van "he".